### PR TITLE
i18n

### PR DESCRIPTION
**Pull request by Drapegnik**
* fetch translations dict from [`Google Drive`](https://docs.google.com/spreadsheets/d/e/2PACX-1vTAexRyfGOsnzvZKvVpPkr8M-l3Ph2vAvBqVu7W_vrPOQ3iUIGg4ZVcOLCeFj-Qg6BowPluH9QO3vXM/pubhtml)
* add `<Text />` component for translation, check out [`storybook`](https://babajka.github.io/babajka-frontend/?knob-pending=false&knob-title=%D0%9D%D0%B5%D0%BA%D0%BE%D1%80%D0%B0%D1%82%D0%BA%D0%B0%20%D0%B0%D0%B1%20%D0%90%D1%85%D0%BC%D0%B0%D1%82%D0%B0%D0%B2%D0%B0%D0%B9&knob-image%20path=static%2Fmedia%2Fphoto5.14563f3a.jpg&knob-author=%D0%97%D0%BE%D1%8F%20%D0%A2%D0%BC%D0%B0%D1%86&knob-touched=false&knob-dict%20key=header.home&knob-className=is-5&knob-subtitle=%D0%9D%D0%B0%D1%81%D1%82%D0%BE%D1%8F%D1%89%D1%83%D1%8E%20%D0%BD%D0%B5%D0%B6%D0%BD%D0%BE%D1%81%D1%82%D1%8C%20%D0%BD%D0%B5%20%D1%81%D0%BF%D1%83%D1%82%D0%B0%D0%B5%D1%88%D1%8C%20%D0%BD%D0%B8%20%D1%81%20%D1%87%D0%B5%D0%BC...&knob-icon=github-alt&knob-locale=en&knob-error=invalid%20email&knob-url=%23&knob-successMessage=correct%20email&knob-article%20link=&knob-label=email&knob-imageClassName=is-3by2&selectedKind=common&selectedStory=Text&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs):
screen shot 2018-04-02 at 18 37 49
* replace old `constants/dictionary.js` with new approach:
screen shot 2018-04-02 at 19 56 43

### CODE REVIEW

**Comment by Drapegnik** [comment]
[i18n. or at least a storage for all labels so that it is easily fixable by non-programmers.](https://trello.com/c/3X1MUxdD/78-i18n-or-at-least-a-storage-for-all-labels-so-that-it-is-easily-fixable-by-non-programmers)

**Comment by Drapegnik** [comment]
[i18n using google spreadsheet](https://trello.com/c/TbjgpTli/163-i18n-using-google-spreadsheet)

**Review by UladBohdan — DISMISSED** [review]
I've tried it locally - works awesome!

**Comment by UladBohdan** [comment]
Crashes if switching to belarusian version when we have an article without belarusian localization.

**Review by UladBohdan — DISMISSED** [review]
Looks great.

File: pages/index.js [hunk 1]
```diff
@@ -46,7 +45,7 @@ class HomePage extends Component {
     return request.populate(ctx, [
       auth.getCurrentUser,
       articlesActions.fetchAll,
-      diaryActions.getByDay.bind(null, DEFAULT_LOCALE, '02', '13'), // temporarily
+      // diaryActions.getByDay.bind(null, DEFAULT_LOCALE, '02', '13'), // FIXME(@tyndria) temporarily
```
Comment (Drapegnik) [inline pages/index.js:48]: @tyndria

**Comment by Drapegnik** [comment]
[to introduce ERROR CODES (as a part of i18n)](https://trello.com/c/R2NAVW1b/157-to-introduce-error-codes-as-a-part-of-i18n)

**Pull request merged**